native/src/routes/CityNotCooperating.tsx

```diff
@@ -77,6 +77,8 @@ const CityNotCooperating = (): ReactElement => {
   const { t } = useTranslation('cityNotCooperating')
   const [isCopied, setIsCopied] = useState<boolean>(false)
   const theme = useTheme()
+  // Only displayed in the Integreat configuration, which has the template
```

Comment:
That is only true as long as we do not add this footer als e.g. for malte, the reason why this can be assumed not null is that it will be only rendered in the Navigator, if cityNotCooperating flag is set. Please also remove this comment.